nextcord/channel.py

```diff
@@ -92,6 +93,62 @@ async def _single_delete_strategy(messages: Iterable[Message]):
         await m.delete()
 
 
+class ForumChannel(abc.GuildChannel, Hashable):
+    __slots__ = (
+
+    )
+    def __init__(self, *, state: ConnectionState, guild: Guild, data: ForumChannelPayload):
+        self._state: ConnectionState = state
+        self.id = int(data['id'])
+        self._type: int = data['type']
+        self._update(guild, data)
+    
+    async def _get_channel(self):
+        return self
+    
+    def _update(self, guild: Guild, data: TextChannelPayload) -> None:
```

Comment:
```suggestion
    def _update(self, guild: Guild, data: ForumChannelPayload) -> None:
```
unsure if some of the keys you use here are in forum